packages/commonwealth/client/scripts/views/modals/update_proposal_status_modal.tsx

```diff
@@ -264,6 +274,10 @@ export const UpdateProposalStatusModal = ({
       throw new Error('Failed to update linked proposals');
     }
 
+    trackAnalytics({
```

Comment:
I haven't looked too deeply into this useBrowserAnalyticsTrack hook, so I could be incorrect here, but does this also send chain and user info to mixpanel? For example as specified in issue:

> With each event, we should always include: community id (where), user id + address id (who)

If it does not, we should add
```
community: thread.chain,
userId: 
addressId: app.user.activeAccount?.address
```

I would argue we don't need to track userId since we have a foreign key constraint from addressId to userId, so we would just be sending and storing duplicate data on mixpanel.

Reply:
Hey @kurtisassad! Thanks for your review!
In response to your question, yes `useBrowserAnalyticsTrack` does send along chain and user info to Mixpanel by default. 

Agreed! I read that statement as sending along user id or address id, not both.